lib-rca/src/main/java/com/facebook/openwifi/librca/inputs/RCAParams.java

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.openwifi.librca.inputs;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/** Define root cause analysis configuration parameters */
+public final class RCAParams {
+
+	/** Look-back window in minutes */
+	public final int detectionWindowMin;
+
+	// KPI calculation parameters
+	/** Minimum acceptable estimated throughput (Mbps) */
+	public final double minEstimatedThroughputMbps;
+	/** Percentile (units are %) of estimated throughputs to use as the KPI */
+	public final double throughputAggregationPercentile;
+	/** Maximum acceptable latency (ms) */
+	public final int maxLatencyThresholdMs;
+	/** Maximum acceptable jitter (ms) */
+	public final int maxJitterThresholdMs;
+	/**
+	 * Maximum acceptable disconnection rate (disconnetions per hour). Note that
+	 * this signifies a rate and the units happen to be per hour - this does not
+	 * signify that every contiguous one-hour period be checked.
+	 */
+	public final int maxDisconnectionRatePerHour;
+
+	// High Level metrics parameters
+	/** Minimum acceptable tx rate (Mbps) */
+	public final double minTxRateMbps;
+	/** Maximum acceptable Packet Error Rate (PER) (units are %) */
+	public final double maxPERPercent;
+	/** Minimum acceptable idle airtime (units are %) */
+	public final double minIdleAirtimePercent;
+	/** Maximum acceptable number of clients for one radio */
+	public final int maxNumClients;
+
+	// Low Level metrics parameters
+	/** Minimum acceptable RSSI (dBm) */
+	public final int minRssidBm;
+	/** Maximum acceptable noise (dBm) */
+	public final int maxNoisedBm;
+	/** Maximum acceptable intf airtime (units are %) */
+	public final double maxIntfAirtimePercent;
+	/** Maximum acceptable number of neighbors */
+	public final int maxNumNeighbors;
+	/** Minimum acceptable client bandwidth (MHz) for non-2G bands / */
+	public final int minClientBandwidthMHz;
+	/** Minimum acceptable Access Point (AP) bandwidth (MHz) for non-2G bands */
+	public final int minApBandwidthMHz;
+	/** Minimum acceptable self airtime ratio (units are %) */
+	public final double minSelfAirtimeRatioPercent;
+	/** Maximum acceptable tx dropped ratio (units are %) */
+	public final double maxTxDroppedRatioPercent;
+
+	/** Default constructor */
+	public RCAParams() {
```

Comment:
Don't you need another constructor? Or remove "final" keyword from everything here?

Reply:
I figured the input would eventually come in as json, so when we parse that, we wouldn't need a separate constructor. (For now, we'd just use the default values for everything.) I also wanted to keep final just so they can be public without adding getters. Thoughts?

Reply:
Can you add that as a comment on the class?